### ISSUE TITLE
level cherry pick solution does not match problem.

### ISSUE THREAD

**Issue by nimbius** [pcottle/learnGitBranching#1143]
answer should be cherry-pick C3 C5 C7, solution shows C3 C4 C7

**Comment by PiotrNowek** [comment]
Hi :)
I found a bug in the `learnGitBranching` project. Here are the details:

**Description:**
In lesson 1 level 3(main)  program except answer C4 in final task, I expected answer C5. The program did not skip to the next lesson 

**Expected Behavior:**
Answer C5 is correct

**Actual Behavior:**
Answer C4 is correct

**Screenshots/Logs:**
![2024-05-24 11_24_47-Learn Git Branching — Mozilla Firefox](https://github.com/pcottle/learnGitBranching/assets/169886540/c59b04ca-fee0-435c-84a1-c71d8bf74096)

Thanks for looking into this!

Peter

**Comment by Conmandouglas** [comment]
The solution is indeed correct, and this is false. Let me explain. So currently, the HEAD is checked out at c1 with main, so anything cherry-picked will continue the branch, as you can see. If you use `git cherry-pick c4` it will add just c4 to the branch. So in this case, when you check the goal image with the **Show Goal** button, or the `goal` command, on the far right you can see the main branch is continued, and what branches to commit. You see at the top, c3, which means that is the first value you enter. Then you see c4, and then c7. This means the solution to this issue would be `git cherry-pick c3 c4 c7` which **is what the solution says**.
In short, this issue should be marked as _resolved_, and the solution is correct as far as I can observe. The solution, according to the code itself, is in the following: `"solutionCommand": "git cherry-pick C3 C4 C7",` which is correct.

**Comment by pcottle** [comment]
Thanks @Conmandouglas for taking a look!
